test/SdkTests/SdkTests.csproj

```diff
@@ -238,6 +238,7 @@
       <TestArgs>$(TestArgs) -html "$(ResultsHtmlPath)" $(TestRunnerAdditionalArguments)</TestArgs>
       <TestArgs>$(TestArgs) &gt; $(ResultsStdOutPath) 2&gt;&amp;1</TestArgs>
       <TestArgs>$(TestArgs) -testList SdkIntegrationTests</TestArgs>
+      <TestArgs>$(TestArgs) -parallel none</TestArgs>
```

Comment:
Why this? I think SDK tests should be able to run in parallel

Reply:
There's a point at which there are too many levels of parallelism:

Arcade runs Test targets in parallel across projects
Each of those run xunit which runs tests in parallel
Each of the tests run msbuild in parallel.

The number of dotnet processes is staggering. And at some point doesn't help because we only have so many CPUs.

We hit the same thing in SDK and did this: https://github.com/dotnet/sdk/blob/650f7d77b8ad7b4eadd181b1276d4a74f0fd3ea6/Directory.Build.props#L26

Before that, I could barely type or use my mouse when running SDK tests. I was experiencing the same thing with core-sdk now that it runs sdk tests.

Now, we are getting rate limit issues with too many concurrent requests.